clients/export/export.go

```diff
@@ -110,3 +110,27 @@ func (c Client) ExportAll(ctx context.Context, params *AllParams) error {
 	}
 	return nil
 }
+
+type StackParams struct {
+	OutParams
+	StackId 	string
+}
+
+func (c Client) ExportStack(ctx context.Context, params *StackParams) error {
+	if params.StackId == "" {
+		return clients.ErrMustSpecifyStack
+	}
+
+	stackId := params.StackId
+
+	exportReq := api.TemplateExport{StackID: &stackId}
+	tmpl, err := c.ExportTemplate(ctx).TemplateExport(exportReq).Execute()
+	if err != nil {
+		return fmt.Errorf("failed to export template: %w", err)
```

Comment:
Would include the ID in the message:
```suggestion
		return fmt.Errorf("failed to export stack %q: %w", err)
```